app/src/main/res/layout/pet_favorites_items.xml

```diff
@@ -19,9 +19,12 @@
         android:layout_toRightOf="@id/petFavImage"
         android:layout_alignParentTop="true"
         android:layout_width="wrap_content"
-        android:text="foo"
+        android:text="Pet Name"
```

Comment:
food for thought, consider adding strings that appear in the UI to our strings.xml file. A good habit to develop early on.